test/native/jest.config.js

```diff
@@ -13,48 +13,45 @@ if ( process.env.TEST_RN_PLATFORM ) {
 	console.log( 'Setting RN platform to: default (' + defaultPlatform + ')' );
 }
 
-const configPath = 'test/native';
-
 const transpiledPackageNames = glob( '../../packages/*/src/index.{js,ts}' ).map(
 	( fileName ) => fileName.split( '/' )[ 3 ]
 );
 
 module.exports = {
 	verbose: true,
-	rootDir: '../../',
+	rootDir: '.',
+	roots: [ '<rootDir>/../..' ],
 	// Automatically clear mock calls and instances between every test.
 	clearMocks: true,
 	preset: 'react-native',
-	setupFiles: [ '<rootDir>/' + configPath + '/setup.js' ],
-	setupFilesAfterEnv: [ '<rootDir>/' + configPath + '/setup-after-env.js' ],
+	setupFiles: [ '<rootDir>/setup.js' ],
+	setupFilesAfterEnv: [ '<rootDir>/setup-after-env.js' ],
```

Comment:
Nice simplification here 👍 

It seems like it's arguable which one is better since now we have to specify a bunch of additional traversing below. `../../`.

Reply:
I had to change the root, and also to introduce a special `babel.config.js` in `packages/e2e-tests`, because of https://github.com/facebook/jest/pull/12689. After that change Jest (namely `babel-jest`) will find `test/native/babel.config.js` only if the `rootDir` is `test/native`. So I had to change it, although it introduces extra `../..` traversals.